include/srtp.h

```diff
@@ -1484,6 +1697,18 @@ srtp_err_status_t srtp_set_debug_module(char *mod_name, int v);
  */
 srtp_err_status_t srtp_list_debug_modules(void);
 
+/**
+ * @brief srtp_get_protect_data_added_length(policy, use_mki, mki_index, is_rtcp, length)
+ *
+ * Determines the length of the amount of data Lib SRTP will add to the 
+ * packet during the protect process. The length is returned in the length parameter
+ *
+ * returns err_status_ok on success, err_status_bad_mki if the MKI index is invalid
+ *
+ */
+srtp_err_status_t srtp_get_protect_data_added_length(const srtp_policy_t *policy, uint32_t use_mki,
```

Comment:
This is more of a coding style.  I don't want to allocate memory that isn't needed.  If you are an appliance (non endpoint) you may have thousands of calls going through you and being terminated.  I know memory is cheap today, but still this is my coding preference.

I understand the push to a session.  That would probably make more sense then a policy.